README.md

```diff
@@ -30,6 +26,17 @@ Check out our NeurIPS 2021 demonstration ["Training Transformers Together"](http
 To learn more about the ideas behind this library,
 see the [full list](https://github.com/learning-at-home/hivemind/tree/refer-to-discord-in-docs#citation) of our papers below.
 
+## Hivemind in Action: Applications and Use Cases
+
+This section lists external projects that leverage hivemind for decentralized training. 
+If you have succesfully trained a model or created a downstream repository with the help of our library, 
+feel free to submit a pull request that adds your project to this list.
+
+* [sahajBERT](https://github.com/tanmoyio/sahajbert) — a collaboratively pretrained ALBERT-xlarge for the Bengali language
```

Comment:
nit: Let's add trailing dots to all list items. The other lists in the readme have them, and the list records are long enough to look a bit strange without the dots.